src/main/java/org/opensearch/jdbc/types/StructType.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.jdbc.types;
+
+import java.sql.Struct;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.LinkedHashMap;
+import java.util.Collections;
+
+
+
+import org.opensearch.jdbc.StructImpl;
+ 
+public class StructType implements TypeHelper<Struct> {
+
+    public static final StructType INSTANCE = new StructType();
+
+    private StructType() {
+
+    }
+
+    @Override
+    public String getTypeName() {
+        return "Struct";
+    }
+
+    @Override
+    public Struct fromValue(Object value, Map<String, Object> conversionParams) {
```

Comment:
Do you know what is the use of second param `conversionParams`?

Reply:
`conversionParams` is an optional parameter that gives the ability to pass in some additional metadata that can be used for creating value, currently used for most date/time types. However, it's not applicable for Struct type and many of the other classes that extend TypeHelper don't use it as well.